src/AutoRest.CSharp/Properties/launchSettings.json

```diff
@@ -268,6 +268,10 @@
       "commandName": "Project",
       "commandLineArgs": "--standalone $(SolutionDir)\\test\\TestProjects\\OperationGroupMappings\\Generated"
     },
+    "PageNextLink": {
```

Comment:
Does this project add something new on top of the existing test-server paging swagger?

Reply:
Those were in autorest.testserver and this felt possibly specific to C#'s handle of x-ms-pagable. I have no strong feeling here, I can push them up if you think that's preferable.